lib/ssl/tls13con.c

```diff
@@ -1611,6 +1611,13 @@ tls13_HandleCertificateRequest(sslSocket *ss, SSL3Opaque *b, PRUint32 length)
     if (rv != SECSuccess)
         goto loser;
 
+    /* We don't support post-handshake client auth, the certificate request
```

Comment:
Instead of doing this, why not just encode a 0 rather than an empty-length array.